lisk-dex-electron/src/Utils.js

```diff
@@ -1,16 +1,89 @@
 // Eventually store all config in singleton class and make it accessible everywhere
-const CryptoAsset = (() => {
-  let assetConfig = { };
+
+const GlobalConfiguration = (() => {
+  let globalConfig = { };
 
   return {
-    getConfig(assetName) {
-      return assetConfig[assetName];
+    getConfig() {
+      return globalConfig;
+    },
+    setConfig(config) {
+      globalConfig = config;
+    },
+    getAppTitle() {
+      return globalConfig.appTitle;
+    },
+    getNotificationDuration() {
+      return globalConfig.notificationDuration;
+    },
+    getRefreshInterval() {
+      return globalConfig.refreshInterval;
+    },
+    getFeedbackLink() {
+      return globalConfig.feedbackLink?.url;
+    },
+    getFeedbackText() {
+      return globalConfig.feedbackLink?.text;
     },
-    setConfig(assets) {
-      assetConfig = assets;
+    getAssets() {
+      return globalConfig.assets && Object.keys(globalConfig.assets);
+    },
+    getAsset(assetName) {
+      return globalConfig.assets && globalConfig.assets[assetName];
+    },
+    getAssetApiUrl(assetName) {
+      return this.getAsset(assetName)?.apiUrl;
     },
     getAssetUnitValue(assetName) {
-      return assetConfig[assetName] && assetConfig[assetName].unitValue;
+      return this.getAsset(assetName)?.unitValue;
+    },
+    getAssetProcessingHeightExpiry(assetName) {
+      return this.getAsset(assetName)?.processingHeightExpiry;
+    },
+    getMarkets() {
+      return globalConfig.markets && Object.keys(globalConfig.markets);
+    },
+    getMarket(market) {
+      return globalConfig.markets && globalConfig.markets[market];
+    },
+    getDefaultActiveMarket() {
+      return this.getMarkets()[0];
+    },
+    getMarketAssets(market) {
+      return this.getMarket(market)?.assets;
+    },
+    getMarketApiUrl(market) {
+      return this.getMarket(market)?.apiUrl;
+    },
+    getMarketPriceHistoryAPI(market) {
+      return this.getMarket(market)?.priceHistoryAPI;
+    },
+    getMarketOptions(market) {
+      return this.getMarket(market)?.marketOptions;
+    },
+    getMarkeVersion(market) {
+      return this.getMarketOptions(market)?.version;
+    },
+    getMarketBaseChain(market) {
+      return this.getMarketOptions(market)?.baseChain;
+    },
+    getMarketPriceDecimalPrecision(market) {
+      return this.getMarketOptions(market)?.priceDecimalPrecision;
+    },
+    getMarketChains(market) {
```

Comment:
`getMarketChains` and `getMarketChain` sound similar but they return very different kinds of data. Maybe we should rename one of these so make the difference clear. Maybe `getMarketChains` can be `getMarketChainNames` or maybe `getMarketChain` can be `getMarketChainOptions`.